src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/Config.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.DotNet.SourceBuild.SmokeTests;
+
+internal static class Config
+{
+    public static string DotNetDirectory { get; } = Environment.GetEnvironmentVariable("DOTNET_DIR") ?? "./.dotnet";
+    public static string DotNetTarballPath { get; } = Environment.GetEnvironmentVariable(DotNetTarballPathEnv) ?? string.Empty;
+    public const string DotNetTarballPathEnv = "DOTNET_TARBALL_PATH";
```

Comment:
Should the other environment variable names also be pulled out into constants?

Reply:
The only reason `DotNetTarballPathEnv` exists as a const is because it is referenced outside of this class.  I didn't see value in  encapsulating the others.  It adds maintenance overhead IMO.  If you see benefit in defining constants let me know.